src/app/index.tsx

```diff
@@ -0,0 +1,125 @@
+import * as React from 'react';
+import { hot } from 'react-hot-loader';
+import './app.scss';
+import * as mac from 'assets/imgs/mac.png';
+import * as windows from 'assets/imgs/windows.png';
+import * as linux from 'assets/imgs/linux.png';
+import * as Secure from 'assets/imgs/icon-shield.svg';
+import * as Offline from 'assets/imgs/icon-offline.svg';
+import * as Yours from 'assets/imgs/icon-hdd.svg';
+import * as OpenSource from 'assets/imgs/icon-github.svg';
+import { Block } from './block';
+import { Platform } from './platform';
+
+const builds = [
+  { name: 'Mac', link: 'https://github.com/MyCryptoHQ/MyCrypto/releases/tag/1.0.0-RC' },
+  { name: 'Windows', link: 'https://github.com/MyCryptoHQ/MyCrypto/releases/tag/1.0.0-RC' },
+  { name: 'Linux', link: 'https://github.com/MyCryptoHQ/MyCrypto/releases/tag/1.0.0-RC' },
+  { name: 'HTML Build', link: 'https://github.com/MyCryptoHQ/MyCrypto/releases/tag/1.0.0-RC' }
+];
+
+let OS: string = '';
+if (navigator.appVersion.includes('Win')) {
+  OS = 'Windows';
+} else if (navigator.appVersion.includes('Mac')) {
+  OS = 'Mac';
```

Comment:
Mac is the line of hardware, MacOS or OSX is the software.

Reply:
It is, personally I prefer 'Download for Mac' instead of 'Download for OSX'. Maybe @dternyak could weigh in on this?

Reply:
OSX has actually been re-branded to macOS.